index.js

```diff
@@ -9,11 +9,11 @@
 module.exports = function base (ALPHABET) {
   var ALPHABET_MAP = {}
   var BASE = ALPHABET.length
-  var LEADER = ALPHABET.charAt(0)
+  var LEADER = ALPHABET[0]
```

Comment:
I don't believe this is guaranteed to work as well as we think?
Can we get some reference to `.charAt` up here first,  and if necessary,  split `ALPHABET` prior?